lime/lime_tabular.py

```diff
@@ -330,3 +332,107 @@ def __data_inverse(self,
             inverse[1:] = self.discretizer.undiscretize(inverse[1:])
         inverse[0] = data_row
         return data, inverse
+
+    def explain_regressor_instance(self, data_row, predict_fn, num_features=10,
```

Comment:
Again, I think there is enough code duplication here that it may be worth just having a parameter either in the init function or in explain_instance, specifying if it's regression or classification.